deps/spidershim/test/value.cc

```diff
@@ -963,3 +963,50 @@ TEST(SpiderShim, Function) {
   EXPECT_TRUE(gen->IsFunction());
   EXPECT_FALSE(genObject->IsFunction());
 }
+
+template <typename TypedArray, int kElementSize>
+static Local<TypedArray> CreateAndCheck(Local<ArrayBuffer> ab,
+                                        int byteOffset, int length) {
+  Local<TypedArray> ta = TypedArray::New(ab, byteOffset, length);
+  EXPECT_EQ(byteOffset, static_cast<int>(ta->ByteOffset()));
+  EXPECT_EQ(length * kElementSize, static_cast<int>(ta->ByteLength()));
+  EXPECT_EQ(ab->GetContents().Data(), ta->Buffer()->GetContents().Data());
+  EXPECT_EQ(ta->ByteOffset(), TypedArray::Cast(*ta->ToObject())->ByteOffset());
+  return ta;
+}
+
+TEST(SpiderShim, ArrayBuffer_NeuteringApi) {
+  // This test is adopted from the V8 ArrayBuffer_NeuteringApi test.
+  V8Engine engine;
+
+  Isolate::Scope isolate_scope(engine.isolate());
+
+  HandleScope handle_scope(engine.isolate());
+  Local<Context> context = Context::New(engine.isolate());
+  Context::Scope context_scope(context);
+  Isolate* isolate = engine.isolate();
+
+  Local<v8::ArrayBuffer> buffer = v8::ArrayBuffer::New(isolate, 1024);
```

Comment:
Looks like you needed to go a global replace.  ;)  I'll fix this myself after merging.